windows/win_package.ps1

```diff
@@ -1237,10 +1237,15 @@ Set-Attr $result "changed" $false;
 
 $path = Get-Attr -obj $params -name path -failifempty $true -resultobj $result
 $name = Get-Attr -obj $params -name name -default $path
-$productid = Get-Attr -obj $params -name productid -failifempty $true -resultobj $result
+$productid = Get-Attr -obj $params -name productid
+if ($productid -eq $null)
+{
+    #Alias added for backwards compat.
+    $productid = Get-Attr -obj $params -name product_id -failifempty $true -resultobj $result
+}
```

Comment:
making note, we might want to add the alias to the Get-Attr function as we do on the python argument spec